README.md

```diff
@@ -734,6 +734,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
   - [Mahaveer Singh Barhath](https://github.com/its-mavi-dev)
   - [Mahesh Pawar](https://github.com/maheshvpawar)
+  - [Mahendra Bishnoi](https://github.com/MahendraBishnoi29)
```

Comment:
```suggestion
  - [Mahendra Bishnoi](https://github.com/MahendraBishnoi29)
  - [Mahesh Pawar](https://github.com/maheshvpawar)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix